doc/api/tls.markdown

```diff
@@ -77,6 +77,51 @@ handshake extensions allowing you:
   * SNI - to use one TLS server for multiple hostnames with different SSL
     certificates.
 
+## Modifying the Default TLS Cipher suite
+
+Node.js is built with a default suite of enabled and disabled TLS ciphers.
+Currently, the default cipher suite is:
+
+    ECDHE-RSA-AES128-GCM-SHA256:
```

Comment:
Yes. That's was copied directly from another location in the same doc and the : is part of the syntax